Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
+
+
+|Scenario |UWP   |Windows |Linux |Mac OS X  |
+|---------|------|--------|------|----------|
```

Comment:
I really like the shape of this.  There's a really interesting issue to solve with the Linux column, and ideas are welcome.  The issues are:
1. We need to identify which Linux distro's have been tested (e.g. ubuntu-14-04)
2. What "works" vs "doesn't work" may differ by Linux distro.

So conceptually we may need to have icons for "supported on some versions" and "tested on some versions".  And then we might include a second table with columns only for Linux, containing only the subset of scenarios affected, and they would get the standard "supported" "not yet tested" "not yet supported" icons